file/analysis_options.yaml

```
@@ -0,0 +1,24 @@
+include: package:lints/recommended.yaml
+
+analyzer:
```

It might be worth to enable stricter analyzer checks, since this is a new package with significantly less code than the other packages.
https://dart.dev/guides/language/analysis-options#enabling-additional-type-checks
```suggestion
analyzer:
  language:
    strict-casts: true
    strict-inference: true
    strict-raw-types: true
```

---

Good catch, done.